store/mongo/datastore_mongo.go

```diff
@@ -368,7 +368,7 @@ const (
 	StorageKeyDeviceDeploymentFinished       = "finished"
 	StorageKeyDeviceDeploymentIsLogAvailable = "log"
 	StorageKeyDeviceDeploymentArtifact       = "image"
-	StorageKeyDeviceDeploymentDeviceType     = "devicetype"
+	StorageKeyDeviceDeploymentRequest        = "request"
```

Comment:
What will happen to the device deployments in progress when we upgrade the deployments service? From what I read, it should just skip the check and proceed as before. Can you please confirm?